5 - months with days/solution_5.rb

```diff
@@ -0,0 +1,27 @@
+require 'date'
+NEGATIVE_NUM = -1
+FIRST_MONTH = 1
+LAST_MONTH = 12
+
+class NewDate
+  def initialize(year)
+    @year = year
+    @months_with_days = generate_months_with_days_count
+  end
+
+  def generate_months_with_days_count
```

Comment:
also this method should be in private section because you only use it inside class

Reply:
@cyberik-dev fixed